docs/direct_mig_requirements.md

```diff
@@ -0,0 +1,26 @@
+# Direct Migration Requirements
+
+Direct Migration is available as a feature from Migration Toolkit for Containers (MTC) release 1.4.0. There are two parts of the Direct Migration - Direct 
+Volume Migration and Direct Image Migration. Direct Migration enables the migration of persistent volumes and internal
+images directly from the source cluster to the destination cluster without an intermediary replication repository (object storage)
+There are some requirements which are to be satisfied before you can use the Direct Migration feature of MTC.
+
+1. The clusters (source as well as destination) involved in the migration should expose their respective internal registries
+for external traffic.
+
+2. The remote source and destination clusters should be able to communicate via the OpenShift routes on port 443.
+
+3. The exposed registry route must be configured in the source and destination MigClusters, this can be done by specifying
+the `spec.exposedRegistryPath` field or via the MTC UI.
+
+4. The source as well the destination clusters must be free of all the `Critical` conditions and be in `Ready` state.
```

Comment:
#4, 5, and 6 look like requirements for all migrations, not just DVM/DIM. 
Can someone confirm?

Reply:
Yes, you are right, 4, 5 and 6 are general requirements for all migrations, I will remove them from this document.